crypt4gh/cli.py

```diff
@@ -16,9 +16,9 @@
 Utility for the cryptographic GA4GH standard, reading from stdin and outputting to stdout.
 
 Usage:
-   {PROG} [-hv] [--log <file>] encrypt [--sk <path>] --recipient_pk <path>
-   {PROG} [-hv] [--log <file>] decrypt [--sk <path>] [--sender_pk <path>] [--range <start-end>]
-   {PROG} [-hv] [--log <file>] reencrypt [--sk <path>] --recipient_pk <path> [--sender_public_key <path>]
+   {PROG} [-hv] [--log <file>] encrypt (--sk <path> | --kms_secret_id <id>) --recipient_pk <path>
```

Comment:
I understand the `OR` by using the parenthesis, but I think you make the `--sk` or `--kms_secret_id` compulsory now.
And this conflicts with the possibility of not mentioning the --sk and have the cli look at the default location (or an environment variable).

How do you suggest we make them work, together with the `--kms` flag.
 By the way, I prefer `--kms-secret-id`

Reply:
Right, ok. I suppose we can just make them all optional then and throw an error in the parser if none of them are available?

Reply:
Also wouldn't the style of `--kms-secret-id` have a different style than the other params like `--sender_public_key`. I'm not fussed either way.

Reply:
Hehe, my bad, I knew there would be somewhere where my comment is inconsistent.
Either way, you're right.

Regarding the optional aspect, I think we can add one line so that docopt takes care of it.
Worth trying.